torch/utils/tensorboard/embedding.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+
+def make_tsv(metadata, save_path, metadata_header=None):
+    if not metadata_header:
+        metadata = [str(x) for x in metadata]
+    else:
+        assert len(metadata_header) == len(metadata[0]), \
+            'len of header must be equal to the number of columns in metadata'
+        metadata = ['\t'.join(str(e) for e in l)
+                    for l in [metadata_header] + metadata]
+    import sys
+    if sys.version_info[0] == 3:
```

Comment:
It's possible you can do this with `six` rather than straight version_info checks, but not required.